src/rollup/RollupAdminLogic.sol

```diff
@@ -255,11 +254,10 @@ contract RollupAdminLogic is RollupCore, IRollupAdmin, DoubleLogicUUPSUpgradeabl
         emit OwnerFunctionCalled(20);
     }
 
-    function forceResolveChallenge(address[] calldata stakerA, address[] calldata stakerB)
-        external
-        override
-        whenPaused
-    {
+    function forceResolveChallenge(
```

Comment:
Is this from a differently configured formatter or did they forget to format?

Reply:
Hmm, I am not sure. I will take a look

Reply:
https://github.com/EspressoSystems/nitro-contracts/pull/24/commits/c1782e80b4ab0f0b3746b06a479fe53450c131dd